rust/cyclotron-node/examples/metrics.ts

```diff
@@ -0,0 +1,21 @@
+const cyclotron = require('../.')
+
+async function main() {
```

Comment:
Feels like this should just be a test to be honest... I know there isn't an existing test setup so not going to push for it here but this should be for sure follow up to setup jest and have tests given how core this sdk is becoming

Reply:
I agree, but yeah, one of those things that keeps slipping down prio